src/components/PermissionsManagement/PermissionsConst.js

```diff
@@ -12,6 +12,28 @@ const getAllSubpermissionKeys = permissions => {
   return keys;
 };
 
+export const generatePermissionLabelKeyMapping = (permissionLabels, start) => {
```

Comment:
```suggestion
export const generatePermissionLabelKeyMap = generatePermissionLabelKeyMapping(permissionLabels, 0);
const generatePermissionLabelKeyMapping = (permissionLabels, start) => {
```
Here's the exported main function instead of the helper.